app/services/auth.server.ts

```diff
@@ -60,8 +60,10 @@ if (process.env.DEV_LOGIN === "true") {
         try {
           const user = await createOrLoginWithDev(secret);
           return user;
-        } catch (e) {
-          console.log(e);
+        } catch (error) {
+          if (error instanceof Error) {
+            throw new Error(error?.message);
```

Comment:
- seems not obvious why you are rethrowing an error
- why not just throw error when rethrowing

Reply:
```
      throw new Error("Wrong code");
```
Could be more descriptive

Reply:
I am very confused by these questions

This errors is when somethiing bad happens in the create account but the code is correct

Reply:
I think we could something like that in the error, "Wrong code" can mean so meany things haha

Reply:
OHHHH, updated the message